File: Web/src/model/UserNotification.js
var User = require("../model/User");
var Notification = require("../model/Notification");

var UserNotification = project.db.define('UserNotification', {
  id: {type: 'serial'},
  isRead: Boolean
});

UserNotification.hasOne('User', User, {field: 'userId'});
UserNotification.hasOne('Notification', Notification, {field: 'notificationId'});

module.exports = UserNotification;

File: Mobile/MyApplication/app/src/main/java/com/example/ooar/discountproject/model/UserNotification.java
package com.example.ooar.discountproject.model;

/**
 * Created by Onur Kuru on 5.3.2016.
 */
public class UserNotification {

    private Integer id;
    private User userId;
    private Notification notificationId;
    private boolean isRead;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public User getUserId() {
        return userId;
    }

    public void setUserId(User userId) {
        this.userId = userId;
    }

    public Notification getNotificationId() {
        return notificationId;
    }

    public void setNotificationId(Notification notificationId) {
        this.notificationId = notificationId;
    }

    public boolean isRead() {
        return isRead;
    }

    public void setIsRead(boolean isRead) {
        this.isRead = isRead;
    }
}
